mud-contracts/world/mud.config.ts

```diff
@@ -392,5 +393,22 @@ export default mudConfig({
       },
       tableIdArgument: true,
     },
+
+    /************************
+     * KillMail module *
+     ************************/
+
+    KillMailTable: {
+      keySchema: {
+        killMailId: "uint256",
+      },
+      valueSchema: {
+        killer: "address",
+        victim: "address",
+        loss: "KillMailLoss",
+        timestamp: "uint256",
```

Comment:
can we rename to make it more intuitive. What timestamp is this ?

Reply:
Yes definitely.

It's the timestamp of when the kill happened in game.

Reply:
Thinking about changing it to `killTimestamp` or `killTime` or something like that.

What do you think about that @0xxlegolas ?

Reply:
Changed to `killTimestamp`.